android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/NativeRadioButtonFactory.java

```diff
@@ -598,7 +598,20 @@ public void run() {
             radioButton.setTextSize(FormUtils.getValueFromSpOrDpOrPx(optionTextSize, this.context));
             radioButton.setText(optionText);
             radioButton.setEnabled(!readOnly);
-            radioButton.setOnCheckedChangeListener(listener);
+
+            // Fix listener bug on older Android versions
+            if (!TextUtils.isEmpty(jsonObject.optString(JsonFormConstants.VALUE)) &&
```

Comment:
Please use `StringUtils.isNotBlank` instead of the `TextUtils` https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StringUtils.html